tests/phpunit/testdata/plugins/test-plugin-enqueued-scripts-in-footer-check-without-errors/footer.js

```diff
@@ -0,0 +1 @@
+console.log( 'test' );
```

Comment:
Do these files need to exist?

Reply:
We enqueue that JS in plugin file so yes.